frontend/app/src/components/history/transactions/TransactionEventAsset.vue

```diff
@@ -22,10 +22,47 @@
         {{ symbol }}
       </div>
     </div>
+    <div
+      v-if="
+        isStakingProtocol &&
+        event.extraData &&
+        event.extraData.asset &&
+        stakedUsdValue
+      "
+      :class="css['total-staked-indicator']"
+      class="pl-6"
```

Comment:
While the component is not too big, maybe it would make sense o extract this to a new component that takes eventData.extraData as a prop.